edx_proctoring/api.py

```diff
@@ -1870,32 +1874,39 @@ def _get_practice_exam_view(exam, context, exam_id, user_id, course_id):
     """
+    user = USER_MODEL.objects.get(id=user_id)
+
     student_view_template = None
 
     attempt = get_exam_attempt(exam_id, user_id)
 
     attempt_status = attempt['status'] if attempt else None
 
-    if not attempt_status:
-        student_view_template = 'practice_exam/entrance.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.started:
-        provider = get_backend_provider(exam)
-        if provider.should_block_access_to_exam_material():
-            student_view_template = 'proctored_exam/error_wrong_browser.html'
-        else:
-            # when we're taking the exam we should not override the view
-            return None
-    elif attempt_status in [ProctoredExamStudentAttemptStatus.created,
-                            ProctoredExamStudentAttemptStatus.download_software_clicked]:
-        student_view_template = 'proctored_exam/instructions.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.ready_to_start:
-        student_view_template = 'proctored_exam/ready_to_start.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.error:
-        student_view_template = 'practice_exam/error.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.submitted:
-        student_view_template = 'practice_exam/submitted.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.ready_to_submit:
-        student_view_template = 'proctored_exam/ready_to_submit.html'
+    active_account = user.is_active
+
+    if not active_account:
+        student_view_template = 'proctored_exam/inactive_account.html'
+    else:
+        if not attempt_status:
+            student_view_template = 'practice_exam/entrance.html'
+        elif attempt_status == ProctoredExamStudentAttemptStatus.started:
+            provider = get_backend_provider(exam)
+            if provider.should_block_access_to_exam_material():
+                student_view_template = 'proctored_exam/error_wrong_browser.html'
+            else:
+                # when we're taking the exam we should not override the view
+                return None
+        elif attempt_status in [ProctoredExamStudentAttemptStatus.created,
+                                ProctoredExamStudentAttemptStatus.download_software_clicked]:
+            student_view_template = 'proctored_exam/instructions.html'
+        elif attempt_status == ProctoredExamStudentAttemptStatus.ready_to_start:
+            student_view_template = 'proctored_exam/ready_to_start.html'
+        elif attempt_status == ProctoredExamStudentAttemptStatus.error:
+            student_view_template = 'practice_exam/error.html'
+        elif attempt_status == ProctoredExamStudentAttemptStatus.submitted:
+            student_view_template = 'practice_exam/submitted.html'
+        elif attempt_status == ProctoredExamStudentAttemptStatus.ready_to_submit:
+            student_view_template = 'proctored_exam/ready_to_submit.html'
```

Comment:
This looks equivalent to a slightly longer chain of `elif`s, which would help to reflect the way in which the change you're making follows the existing pattern. The added indentation emphasizes the case you're adding unnecessarily.